components/sections/Header.jsx

```diff
@@ -0,0 +1,28 @@
+import CustomLink from "../ui/CustomLink";
+
+export default function Header() {
+    return (
+        <section className="mt-6.6">
+            {/* header  */}
+            <div className="flex gap-5.5 flex-col text-2.5xl font-normal leading-7 ">
+                {/* header log */}
+                <div className="">
+                    Ape Unit
+                </div>
+                {/* header content */}
+                <div className="">
+                    Unit➇ is a pioneering technology and marketing 
+                    services company that creates end-to-end digital
+                    experiences for brands such as 
+                    <span className="text-primary-300">
+                        {" "}Google, Philips, Audi, Twitch, Patagonia, eBay {" "}
+                    </span> 
+                    and more. Its team
+                    of 2,500+ digital specialists across 30+ locations
+                    on 5 continents delivers pioneering work on a global
+                    scale with a boutique culture. <span> <CustomLink text="Contact us" styles={""} /> </span>
+                </div>
```

Comment:
Do we need to explicitly pass the styles as empty string in the custom link component if there are no custom styles?  I think it would look cleaner if it is removed as done to the `Wrapper` section

Reply:
No actually, and I don't see any wrapper in my component

Reply:
I was talking about the way we didn't add styles prop in the Wrapper component ( the one we used in `_app.js`)